app/models/zillow.rb

```diff
@@ -19,22 +19,29 @@ def initialize(attrs)
     @type         = attrs["searchresults"]["response"]["results"]["result"]["useCode"]
     @neighborhood = attrs["searchresults"]["response"]["results"]["result"]["localRealEstate"]["region"]["name"]
     @sqft         = attrs["searchresults"]["response"]["results"]["result"]["finishedSqFt"]
-  end
+    @rent         = attrs["searchresults"]["response"]["results"]["result"]["rentZestimate"]
+    @sold_price   = attrs["searchresults"]["response"]["results"]["result"]["lastSoldPrice"]["__content__"]
+    @sold_date    = attrs["searchresults"]["response"]["results"]["result"]["lastSoldDate"]
```

Comment:
When I do a search for my house (4318 4th Ave NE) I get an error on line 23 because it doesn't have the `lastSoldPrice`, could you wrap that in a conditional?